src/views/credits/credits.jsx

```diff
@@ -257,6 +257,7 @@ var Credits = React.createClass({
                 <p>
                     <a href="http://www.nsf.gov/">National Science Foundation</a>,
                     <a href="http://www.scratchfoundation.org/"> Scratch Foundation</a>,
+                    <a href="https:/www.siegelendowment.org"> Siegel Family Endowment </a>
```

Comment:
Please add a comma at the end of the line and remove the space before the closing `</a>`